crates/common/tedge_config_macros/macro/Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = { workspace = true }
 proc-macro = true
 
 [dependencies]
-tedge_config_macros-impl = { workspace = true }
-syn = { workspace = true }
-quote = { workspace = true }
 proc-macro2 = { workspace = true }
+quote = { workspace = true }
```

Comment:
Looks like the sorting logic of `cargo-sort` was really broken, even for obvious things like this. Glad we replaced it with something more sensible.